### ISSUE TITLE
XML errors fail gradle build

### ISSUE THREAD

**Issue by icasdri** [arpitkh96/AmazeFileManager#301]
Invocation of `gradle build` or `gradle assembleFdroid` fails due to XML errors in two places.

* In [`src/main/res/values-bn/strings.xml`](https://github.com/arpitkh96/AmazeFileManager/blob/master/src/main/res/values-bn/strings.xml)
    * On line 35 and 37 (in the declaration of the `<resources>` tag), the space characters at the start of those lines do not appear to be actual space characters, but rather some other character that's not rendering. The XML parser chokes on these with 

        ```
        Element type "resources" must be followed by either attribute specifications, ">" or "/>".
        ```
    * On line 77 (inside the `<string>` "copyingapk"), there is an unescaped XML sequence `<Folder Name>` which the XML parser chokes on with 

        ```
        Attribute name "Name" associated with an element type "Folder" must be followed by the ' = ' character
        ```

* In [`src/main/res/xml/preferences.xml`](https://github.com/arpitkh96/AmazeFileManager/blob/master/src/main/res/xml/preferences.xml)
    * The preference on line 171 refers to a non-existent string value `@string/brazilian_portuguese_translation_title`. Compilation attempt results in 

        ```
        AAPT: No resource found that matches the given name (at 'title' with value '@string/brazilian_portuguese_translation_title')
        ```

**Comment by proninyaroslav** [comment]
Fixed in #303

**Comment by icasdri** [comment]
Thanks!
